Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_016.StripeApiKey.sarif

```diff
@@ -27,22 +27,6 @@
                 }
               },
               "helpUri": "https://github.com/microsoft/sarif-pattern-matcher"
-            },
-            {
-              "id": "SEC101/019",
-              "name": "DoNotExposePlaintextSecrets/PicaticApiKey",
```

Comment:
>PicaticApiKey [](start = 51, length = 13)

Good fix!

Reply:
we are awesome! :smile: 

---
In reply to: [591968522](https://github.com/microsoft/sarif-pattern-matcher/pull/283#discussion_r591968522) [](ancestors = 591968522)